docs/4.x/tests-github

```diff
@@ -0,0 +1,31 @@
+---
+category: Develop
+previous: tests-js-tracker
+next: tests-troubleshooting
+---
+# Running tests/builds on GitHub Action
+
+[Github Action](https://github.com/features/actions) is a continuous integration tool that will run tests/builds for a GitHub repository every time commits are pushed.
+
+## Matomo on GitHub Action 
+
+Matomo uses GitHub Action to automatically run its build on every commit (for every branch and pull request).
+
+Each developer is responsible to keep the build green.
```

Comment:
We should maybe explain that a bit more in detail and also mention that we have action running on various triggers.
Those action that are running for commits / pull requests are the ones that needs to be kept green